Fretboard.h

```diff
@@ -0,0 +1,33 @@
+/******************************************************************
+    Fretboard.h
+
+    structure to encapsulate a guitar fretboard
+******************************************************************/
+#pragma once
+#include <iostream>
```

Comment:
you aren't using anything from iostream in this header file.
don't include what you don't use.
- [ ] revise

Reply:
copy paste error fixed.